sns/parity_math.c

```diff
@@ -1563,21 +1614,27 @@ M0_INTERNAL int m0_sns_ir_failure_register(struct m0_bufvec *recov_addr,
 	if (is_data(ir, failed_index))
 		M0_CNT_INC(ir->si_failed_data_nr);
 #endif /* ISAL_ENCODE_ENABLED */
-	return (ir->si_alive_nr < ir->si_data_nr) ? M0_ERR(-EDQUOT) : 0;
+
+	return (ir->si_alive_nr < ir->si_data_nr) ? M0_ERR(-EDQUOT) : M0_RC(0);
 }
 
 M0_INTERNAL int m0_sns_ir_mat_compute(struct m0_sns_ir *ir)
 {
 	int			ret = 0;
 	uint32_t		i;
 	uint32_t		j;
+	uint32_t		unit_count;
 	struct m0_sns_ir_block *blocks;
 
+	M0_ENTRY("ir=%p", ir);
 	M0_PRE(ir != NULL);
 
 	blocks = ir->si_blocks;
+	unit_count = block_count(ir);
```

Comment:
block_count() is a very misleading name, or ambiguous name.
And it is even more misleading or ambiguous when you use another name "unit_count" here.

It is N+K.  Maybe we can rename it to,
unit_width?
Maybe some other name.

Reply:
Done. Renamed function block_count() to ir_blocks_count(). Variable unit_count is renamed to total_blocks_nr.